commons/src/test/java/com/powsybl/commons/config/ComponentDefaultConfigTest.java

```diff
@@ -31,6 +33,10 @@ interface A {
     static class B implements A {
     }
 
+    static class C implements A {
+        C(String a, String b) {}
```

Comment:
To fix the checkstyle issue:

```suggestion
        C(String a, String b) { }
```